DEPENDENCIES.md

```diff
@@ -14,7 +14,7 @@
 | libclang-dev     | 7.1              |    Linux |                             Yes (JIT) |
 | libnuma-dev      | any              |    Linux |                            Yes (numa) |
 | libnuma1         | any              |    Linux |                            Yes (numa) |
-| llvm             | any              |    All   |                 Yes (code sanitizers) |
+| llvm             | <= 7.0.1         |    All   |                 Yes (code sanitizers) |
```

Comment:
<= ?

Reply:
Support up to version `7.0.1`. 
Is there also a minimum version?

Reply:
Oh! I wanted to refer to the compiler frontend, which is `clang`, not `llvm`.  I'll remove this change.

Reply:
Ah, for JIT, this makes sense.